packages/cli/src/config/webpack.config.common.js

```diff
@@ -51,6 +51,17 @@ module.exports = ({ config, context }) => {
     }
   ];
 
+  const customHooks = Object.assign({}, ...config.plugins
+    .filter((plugin) => plugin.type === 'hook')
```

Comment:
Can we call these webpackhooks? hooks in general we may have more of typess of them I'm not sure, maybe we can change that later.  when I saw template hooks, I thought it would have to do with page-templates not index.

Reply:
So maybe there are two things here
1. `hooks` are the API, so other plugin types would be handled somewhere else because they would be doing a different thing specific to their own API.
1. The name `hooks` / Template Hooks may not be the most direct / semantic name, so let think of something else we could try and call it instead.

Reply:
What about calling the plugin type _index_ instead?